src/dev-app/_color-api-back-compat.scss

```diff
@@ -0,0 +1,96 @@
+@use '@angular/material' as mat;
+
+@mixin color-variant-styles($theme, $color-variant) {
+  @include mat.option-color($theme, $color-variant: $color-variant);
```

Comment:
Do we want some validation here so people don't pass in random names as the variant?

Reply:
It does do some validation deeper down somewhere. It'll yell at you if you pass something invalid.